<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>650</width>
    <height>704</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>650</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="array_size_label">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Array Size:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spin_box">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>5000</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="create_array_button">
       <property name="text">
        <string>Create Array</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QCheckBox" name="bubble_sort_checkbox">
       <property name="text">
        <string>Bubble Sort</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="insertion_sort_checkbox">
       <property name="text">
        <string>Insertion Sort</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="merge_sort_checkbox">
       <property name="text">
        <string>Merge Sort</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QCheckBox" name="quick_sort_checkbox">
       <property name="text">
        <string>Quick Sort</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="heap_sort_checkbox">
       <property name="text">
        <string>Heap Sort</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="counting_sort_checkbox">
       <property name="text">
        <string>Counting Sort</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QCheckBox" name="radix_sort_checkbox">
       <property name="text">
        <string>Radix Sort</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="bucket_sort_checkbox">
       <property name="text">
        <string>Bucket Sort</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="quick_select_checkbox">
       <property name="text">
        <string>Quick Select</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGraphicsView" name="graph_view">
     <property name="maximumSize">
      <size>
       <width>632</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="text_view">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>571</width>
       <height>41</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>632</width>
       <height>120</height>
      </size>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="start_button">
       <property name="text">
        <string>Start</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
